assets/components/lvms/topolvm-node-metrics_v1_service.yaml

```diff
@@ -14,6 +14,6 @@ spec:
     protocol: TCP
     targetPort: 8080
   selector:
-    app.kubernetes.io/name: topolvm-node
+    app.kubernetes.io/component: topolvm-node
```

Comment:
Is this something we had missed in manual rebase that was found by the automation?

Reply:
Is it necessary to backport this fix to 4.12 release?

Reply:
no, the previous version was also pulled from the operator bundle, so this change will have come from upstream topolvm